packages/api/src/external/fhir/document/index.ts

```diff
@@ -194,22 +199,25 @@ function convertToFHIRResource(resource: Contained, patientId: string): Resource
       gender: convertCWGenderToFHIR(resource.gender?.coding),
     };
   } else {
-    capture.message(`New Resource type on toFHIR conversion`, {
-      extra: {
-        context: `toFHIR.convertToFHIRResource`,
-        resourceType: resource.resourceType,
-        resource,
-        patientId,
-      },
-    });
+    capture.message(
+      `New Resource type on toFHIR conversion - might need to handle in CW doc ref mapping`,
+      {
+        extra: {
+          context: `toFHIR.convertToFHIRResource`,
+          resourceType: resource.resourceType,
+          resource,
+          patientId,
+        },
+      }
+    );
   }
 }
 
 /**
- * Converts a CW name to a string
+ * Converts a CW name to a string.
  *
- * @param name
- * @returns
+ * @param name One of the CW name types.
+ * @returns a space-separated name string.
```

Comment:
nit: in all places - the funcs don't always return the string/HumanName[]/etc, they can return undefined/null so we might was well mention it. `a space-separated name string; otherwise undefined if a valid name cannot be determined`

note this is really NBD for these small funcs, but if we're going to be documenting code, we might as well do it properly - especially if we're setting the precedence for documenting larger funcs